packages/vuetify/src/framework.ts

```diff
@@ -14,59 +19,88 @@ import {
 // Services
 import * as services from './services'
 
-export default class Vuetify {
-  static install = install
+export const VuetifySymbol = Symbol.for('vuetify')
 
-  static installed = false
+export function useVuetify () {
+  const vuetify = inject(VuetifySymbol)
+
+  /* istanbul ignore if */
+  if (!vuetify) {
+    warn('Vuetify has not been installed on this app')
+  }
 
-  static version = __VUETIFY_VERSION__
+  return vuetify
+}
+
+export const version = __VUETIFY_VERSION__
+
+export default class Vuetify {
+  public defaultPreset = Preset
```

Comment:
is there a reason this is saved on the instance instead of just used as imported?

Reply:
Fair assessment.